geomstats/geometry/dirichlet_distributions.py

```diff
@@ -84,6 +84,25 @@ def random_point(self, n_samples=1, bound=5.):
         size = (self.dim,) if n_samples == 1 else (n_samples, self.dim)
         return bound * gs.random.rand(*size)
 
+    def projection(self, point, atol=gs.atol):
+        """Project a point in ambient space to the open set.
+
+        The last coordinate is floored to `gs.atol` if it is negative.
+
+        Parameters
+        ----------
+        point : array-like, shape=[..., dim_embedding]
+            Point in ambient space.
+        atol : float
+            Tolerance to evaluate positivity.
+
+        Returns
+        -------
+        projected : array-like, shape=[..., dim_embedding]
```

Comment:
Shouldn't the projected point have dimension ```dim``` instead of ```dim_embedding```?

Reply:
Yes you're right, because we're in an `OpenSet` :)